test/smoke/package/src/localization.test.ts

```diff
@@ -21,20 +20,21 @@ export function setup() {
         });
 
         it("Test localization", async function () {
-            app = await runVSCode(RNworkspacePath, "ru");
-            console.log("Localization test: Starting packager");
-            await app.workbench.quickaccess.runCommand(startPackagerCommand);
-            await sleep(10 * 1000);
-            if (process.env.REACT_NATIVE_TOOLS_LOGS_DIR) {
+            try{
+                app = await vscodeManager.runVSCode(workspace, "LocalizationTest", "ru");
+                console.log("Localization test: Starting packager");
+                await app.workbench.quickaccess.runCommand(startPackagerCommand);
+                await sleep(10 * 1000);
                 console.log(`Localization test: Search for '${packagerStartedCheck}' string output`);
-                const found = findStringInFile(path.join(process.env.REACT_NATIVE_TOOLS_LOGS_DIR, SmokeTestsConstants.ReactNativeLogFileName), packagerStartedCheck);
+                const found = vscodeManager.findStringInLogs(packagerStartedCheck, SmokeTestsConstants.ReactNativeLogFileName);
```

Comment:
Can it be better to implement findStringInLogs function as part basic util functions class instead of VSCodeManager?